config/21_regions_EU11/scenario_config_EU.csv

```diff
@@ -0,0 +1,7 @@
+title;start;.description;regionmapping;extramappings_historic;CES_parameters;optimization;c_testOneRegi_region;cm_nash_mode;subsidizeLearning;trade;c_ccsinjecratescen;cm_bioprod_regi_lim;cm_import_tax;cm_1stgen_phaseout;c_agricult_base_shift;cm_altFeEmiFac;cm_CESMkup_build;cm_CESMkup_ind;cm_rcp_scen;cm_iterative_target_adj;c_budgetCO2from2020;carbonprice;cm_co2_tax_2020;cm_peakBudgYr;cm_taxCO2inc_after_peakBudgYr;cm_CO2priceRegConvEndYr;regipol;cm_emiMktTarget;cm_NucRegiPol;cm_CoalRegiPol;capitalMarket;industry;buildings;transport;cm_EDGEtr_scen;cm_emiscen;cm_POPscen;cm_GDPscen;cm_demScen;cm_startyear;path_gdx;path_gdx_ref
```

Comment:
I would remove this scenario config file from this PR.
It has being agreed that we shouldn't create more scenario config files in the subfolder 21_regions_EU11.
Also, the default scenario config for EU21 should be added to the default REMIND scenario config file and support the default budget runs supported in REMIND.
We can get together to better define this scenario config switches in an smaller group before adding it to the code.

Reply:
done